src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.spec.ts

```diff
@@ -53,13 +58,16 @@ describe('ConstraintsPanelComponent', () => {
         // Estimated cost in $ per acre
         estimatedCost: ['', [Validators.min(0), Validators.required]],
         // Max cost of treatment for entire planning area
-        maxCost: ['', Validators.min(0)],
+        maxCost: [{ value: '', disabled: true }, Validators.min(0)],
```

Comment:
do we still want the validator?

Reply:
We would still want Validators.min(0) if we want to make sure maxCost is only ever positive. It doesn't mean that the input is required, just that it has to be at least 0. 

Another question would be is there a different min or an additional maximum we want to impose on these values

Reply:
I don't think we need a max on maxCost yet.  If a user puts in a ginormous number, it's like not setting a maximum.

Thinking more about it, let's leave this as is.